app/views/admin/organizations/show.html.slim

```diff
@@ -2,8 +2,11 @@
 
 section.coop.details
 
-  section.block.coop-fields  class=(@org.valid? ? 'show-view' : 'edit-view')
+  section.block.coop-fields class=(@org.valid? ? 'show-view' : 'edit-view')
     .show-actions
+      a href=admin_project_logs_path(org: @org.id)
+        = t 'common.view_logs'
```

Comment:
Please use parens to be consistent with rest of page. I know we don't have a style guide for this but there is a clear precedent on this particular page...